app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from typing import List
+
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: List[Player]) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: List[Elf]) -> None:
+    [elf.play_elf_song() for elf in elves]
```

Comment:
there is no need in the list comprehension as you don't need to store the results of the function. a simple loop is enough